CONTRIBUTING.md

```diff
@@ -38,14 +39,40 @@ Bug reports and specific feature requests are best discussed in an issue on the
 
 ### Code or document change contributions (GitHub User Interface)
 
-Changes can be made from within the GitHub user interface. As a non-maintainer, GitHub will create a fork under your name and ask that you create a pull request. The pull request lets the repository maintainers know of your change and provides a space to discuss the change.
+Changes can be made from within the GitHub user interface. As a non-maintainer, GitHub will create a fork under your name and ask that you create a pull request. The pull request lets the repository maintainers know of your change and provides a space to discuss the change.  
+
+The [CHAOSS Charter](https://github.com/chaoss/governance/blob/master/project-charter.md) requires that contributions
+are accompanied by a [Developer Certificate of Origin](http://developercertificate.org) sign-off.  
+For ensuring it, a bot checks all incoming commits.  
+
+A DCO sign-off is accomplished by writing into the commit comment field:  
 
-Every commit needs a [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
-A DCO sign-off is accomplished by writing into the commit comment field:
 `Signed-off-by: Your Name <YourName@example.org>`   
+  
 It should be noted that the angled brackets `<` and `>` need to be included with the email.
 
-This process can be automated by using a browser plugin like [scottrigby/dco-gh-ui](https://github.com/scottrigby/dco-gh-ui). This plugin is non-intrusive and works only on GitHub. You would need to input your registered GitHub email address in the email field.
+This can be automated by using a browser plugin like
+[DCO GitHub UI](https://github.com/scottrigby/dco-gh-ui).
+
+#### Steps to use the DCO browser plugin
+The  [DCO browser plugin](https://github.com/scottrigby/dco-gh-ui) is a handy tool to automatically sign commits created using GitHub. 
+To enable this plugin: 
+
+- Go to the plugin page on the [chrome web store](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo).
+- Alternatively, you could go to the [firefox addon page](https://addons.mozilla.org/en-US/firefox/addon/scott-rigby/) to add the extension to your browser.
+- Once you add the extension, right click on the extension in the toolbar of your browser and select `Options`. 
+- A dialog box will open up as shown below. Fill in your GitHub name (not the handle) and email-id. 
+
+ ![Screenshot of settings for DCO GitHub UI](https://user-images.githubusercontent.com/31214064/55411911-194c8500-5584-11e9-8b56-c8f94b6fa213.png)
```

Comment:
Maybe remove image?

I'm not a fan of images embedded in a CONTRIBUTING.md file for two reasons:
- the file when seen in an editor will not load pictures
- the image is not hosted on the repo, nor does it belong there

Reply:
Hello @GeorgLink I have removed the pictures. 

Thanks.